Core/AppRuntime/Include/Babylon/AppRuntime.h

```diff
@@ -15,7 +16,7 @@ namespace Babylon
         AppRuntime();
         ~AppRuntime();
 
-        void Suspend();
+        std::future<void> Suspend();
```

Comment:
Who uses the std::future?

Reply:
At present no one. Its inclusion comes from your observation back when suspend is actually an asynchronous operation and some mechanism should be provided for determining when the suspension has truly happened. This future provides such a mechanism without exposing Arcana.

Reply:
I did test it -- it was used by the old AppLifecycle code that this change used to be in a PR with -- but I'm fine with leaving it out and putting it back in when/if we need it. It won't be a hard change to make later.